src/pages/GapsPatternsPage.scss

```diff
@@ -0,0 +1,6 @@
+.tooltip-style {
+    background: white;
+    padding: 5px;
+    border: 1px solid #ccc;
+    border-radius: 5px;
+}
```

Comment:
@shootermv we changed the tooltip - I'm not sure this code is relevant anymore

Reply:
wait - we use two tooltips (mui and rechart tooltips), so it might be fine

Reply:
is there a way to check it to be sure?

Reply:
it's good